### DISCUSSION TITLE
Why GitHub Wiki?

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [ThioJoe/YT-Spammer-Purge#799] [Ideas]
## Problems

Why is the documentation using **GitHub Wiki** instead
of a `Documentation` folder when:

- It's a lot more difficult to contribute to GitHub Wikis

- It's problematic to change links

  *For example changing a page name / url in the wiki*
  *or file in the repo requires two separate changes in*
  *each of them.*

- It doesn't allow for shorter relative links 
  that also work on locally cloned version

  *You always have to specify absolute*
  *links to resources from the repo.*

- It doesn't allow for folder structures,
  making it messy.

<br>

## Solution

A simple `/Documentation/` folder in the repo.

Doesn't make the root folder substantially longer.

Additionally the content can be sorted into
subfolders such as `/Install/`.

@ThioJoe If you don't mind this change,
I would work on it and submit a PR.

### Steps

The steps I would take:

1. Create docs folder with exact same content as the wiki
  
  *For reference ( changes )*

2. Sort into subfolders

3. Do basic refactors like:
  - Convert raw links to referenced ones
  - Adjust spacing
  - Fix spelling

In subsequent PRs I would like to address more
content oriented changes like rewording / rewriting
paragraphs as well as some restructuring.

**Comment by Firecul** [comment]
It would also allow for making an [mkdocs](https://github.com/mkdocs/mkdocs) page for example.

**Comment by ElectronicsArchiver** [comment]
Also, in the case that the one added `/Documentation/` folder
would be too much, would could utilize a dedicated branch for
documentation instead~

**Comment by TechStudent10** [comment]
I think that the Wiki is fine as is, but I would much prefer to have a `docs` folder (not `Documentation`) that contains how to use the program, but even deep-dives as to how a certain feature works, but that's optional.
